lib/nexmo_developer/app/views/layouts/partials/_header.html.erb

```diff
@@ -3,6 +3,7 @@
 
 <div class="Adp-header">
   <header id="header" class="Adp-header__main Vlt-M-plus">
+  <input type="hidden" id="adobe_launch_primary_category" value="<%=ENV['ADOBE_LAUNCH_PRIMARY_CATEGORY']%>">
```

Comment:
Can we use environment variables in JavaScript rather than going through ruby & HTML?